src/ReaderScreen/styles/index.js

```diff
@@ -46,15 +46,13 @@ const createStyles = (theme) => ({
   },
   container: {
     borderRadius: theme.radius.lg + theme.spacing.lg,
-    position: "absolute",
-    bottom: 5,
-    left: 0,
-    right: 0,
-    // paddingTop: 0,
+    width: "100%",
+    zIndex: 100,
     paddingHorizontal: theme.spacing.xl,
     paddingVertical: theme.spacing.sm,
     overflow: "hidden",
     backgroundColor: theme.colors.primary,
+    elevation: 12,
```

Comment:
Afaik, this isn't supported in iOS, do you have separate styles in place to have same effect for ios devices?